lms/djangoapps/grades/api/v1/views.py

```diff
@@ -577,73 +541,72 @@ class GradebookView(GradeViewMixin, PaginatedAPIView):
-        # Fields we may not need:
-        # ['are_grades_published', 'auto_grade', 'comment', 'detail', 'is_ag', 'is_average', 'is_manually_graded']
-        # Some fields should be renamed:
+        # TODO: https://openedx.atlassian.net/browse/EDUCATOR-3559 -- Some fields should be renamed:
         # 'displayed_value' should maybe be 'description_percent'
         # 'grade_description' should be 'description_ratio'
 
-        label_finder = SubsectionLabelFinder(course_grade)
         default_labeler = get_default_short_labeler(course)
 
-        for chapter_location, section_data in course_grade.chapter_grades.items():
-            for subsection_grade in section_data['sections']:
-                default_short_label = default_labeler(subsection_grade.format)
-                breakdown.append({
-                    'are_grades_published': True,
-                    'auto_grade': False,
-                    'category': subsection_grade.format,
-                    'chapter_name': section_data['display_name'],
-                    'comment': '',
-                    'detail': '',
-                    'displayed_value': '{:.2f}'.format(subsection_grade.percent_graded),
-                    'is_graded': subsection_grade.graded,
-                    'grade_description': '({earned:.2f}/{possible:.2f})'.format(
-                        earned=subsection_grade.graded_total.earned,
-                        possible=subsection_grade.graded_total.possible,
-                    ),
-                    'is_ag': False,
-                    'is_average': False,
-                    'is_manually_graded': False,
-                    'label': label_finder.get_label(subsection_grade.display_name) or default_short_label,
-                    'letter_grade': course_grade.letter_grade,
-                    'module_id': text_type(subsection_grade.location),
-                    'percent': subsection_grade.percent_graded,
-                    'score_earned': subsection_grade.graded_total.earned,
-                    'score_possible': subsection_grade.graded_total.possible,
-                    'section_block_id': text_type(chapter_location),
-                    'subsection_name': subsection_grade.display_name,
-                })
-                if subsection_grade.graded and subsection_grade.graded_total.possible > 0:
-                    aggregates[subsection_grade.format]['score_earned'] += subsection_grade.graded_total.earned
-                    aggregates[subsection_grade.format]['score_possible'] += subsection_grade.graded_total.possible
+        for subsection in graded_subsections:
+            subsection_grade = course_grade.subsection_grade(subsection.location)
+            short_label = default_labeler(subsection_grade.format)
+
+            graded_description = 'Not Attempted'
+            score_earned = 0
+            score_possible = 0
+            if subsection_grade.attempted_graded:
```

Comment:
#NIT Could you add some comment here to speak about the reason we chose this treatment?